### DISCUSSION TITLE
Do you have plans to migrate the codebase from Cairo v0 to Cairo v1.0?

### DISCUSSION THREAD

**Discussion by 0x180db** [keep-starknet-strange/garaga#68] [General]
Hi guys! It looks like there's a lot of work to be done. Can you share your plans?

**Comment by feltroidprime** [comment]
@0x180db Hello! Thank you for your interest. We indeed want to convert the logic to Cairo 1, however the current focus is auditing emulated field security (in progress now) and write builtins for the core add,sub & mul functions. 
If you are interested in helping re-writing the lib to Cairo 1, please join the Garaga telegram group and ping me there https://t.me/GaragaPairingCairo , we are very happy to welcome contributors for this migration. 
Thanks and sorry for the late answer, I didn't get a notification for this github discussion.

**Comment by feltroidprime** [comment]
We support both now !
